Dockerfile

```diff
@@ -1,13 +1,31 @@
-FROM alpine
+FROM ruby:2.6-alpine
 
-RUN apk update
-RUN apk add git openssh-client curl jq
+RUN apk add \
```

Comment:
We could make smaller image by adding `--no-cache` when installing the apk packages

Reply:
Done.